test/sql/new_feature_test.cpp

```diff
@@ -0,0 +1,198 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// aggregate_sql_test.cpp
+//
+// Identification: test/sql/aggregate_sql_test.cpp
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include <memory>
+
+#include "sql/testing_sql_util.h"
+#include "catalog/catalog.h"
+#include "common/harness.h"
+#include "executor/create_executor.h"
+#include "optimizer/simple_optimizer.h"
+#include "planner/create_plan.h"
+
+
+namespace peloton {
+  namespace test {
+
+    class NewFeatureTest : public PelotonTest {};
+
+    void CreateAndLoadTable() {
+      // Create a table first
+      LOG_INFO("Creating a table...");
+      LOG_INFO(
+      "Query: create table a(id int, value int)");
+
+      TestingSQLUtil::ExecuteSQLQuery(
+      "create table a(id int, value int);");
+
+      LOG_INFO("Table created!");
+
+      // Insert multiple tuples into table
+      LOG_INFO("Inserting a tuple...");
+      LOG_INFO(
+              "Query: Iinsert into a values(1, 1)");
+      LOG_INFO(
+              "Query: Iinsert into a values(2, null)");
+      LOG_INFO(
+              "Query: Iinsert into a values(3, null)");
+      LOG_INFO(
+              "Query: Iinsert into a values(4, 4)");
+
+      TestingSQLUtil::ExecuteSQLQuery("insert into a values(1, 1);");
+      TestingSQLUtil::ExecuteSQLQuery("insert into a values(2, null);");
+      TestingSQLUtil::ExecuteSQLQuery("insert into a values(3, null);");
+      TestingSQLUtil::ExecuteSQLQuery("insert into a values(4, 4);");
+
+      LOG_INFO("Tuple inserted!");
+    }
+      TEST_F(NewFeatureTest, WhereInTest) {
```

Comment:
Do we now support IN with multiple values (or when the value is a set)? How about when the value is an expression? If so, these test cases should be added.